lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -112,12 +112,24 @@ export default function LMSFilePicker({
           path: listFilesApi.path,
         })
       );
+
+      /**
+       * FIXME: This is a temporary fix for the updated file API response for
+       * BlackBoard files. A File returned by the BlackBoard files API may be
+       * either a `File` or a `Folder`. Until `FileList` and this component
+       * have has been updated to render and handle `Folder`-type objects,
+       * filter them out of the file list: i.e. hide them.
+       */
+      const filteredFiles = files.filter(
+        file => !file.type || file.type === 'File'
+      );
```

Comment:
I know this is only intended as a temporary change, but it shouldn't be too hard to add a small test. This could just be a slight variant of the `fetches files when the dialog first appears` one.

Reply:
Um, derp, oversight. Let me see if a test is easy enough to drop in.